content/en/blog/_posts/Monitoring Kubernetes workloads: the sidecar pattern

```diff
@@ -0,0 +1,174 @@
+    value: wss://sensu-backend-0.sensu.sensu-system.svc.cluster.local:8081 wss://sensu-backend-1.sensu.sensu-system.svc.cluster.local:8081 wss://sensu-backend-2.sensu.sensu-system.svc.cluster.local:8081
+  - name: SENSU_NAMESPACE
+    value: webinar
+  - name: SENSU_SUBSCRIPTIONS
+    value: linux kubernetes nginx
+  - name: SENSU_DEREGISTER
+    value: "true"
+  - name: SENSU_STATSD_EVENT_HANDLERS
+    value: statsd
+```
+
+Now that you have this configuration in place, you can scale resources easily. 
+
+```
+kubectl --namespace webinar scale deployment nginx --replicas 10
+```
+
+Before we configure the monitoring check, I’ll provide some background on Sensu’s architecture and how the services communicate. The Sensu agent communicates with the Sensu backend in a pub-sub model, and the agent only requires outbound network access so we don’t have to open ports in the agent or on the pod to get access to them. Agents have TLS-encrypted web sockets in Sensu Go, and they maintain this connection with the Sensu backend, with built-in support for high availability. 
+
+When the agent connects, it self-describes its role. In the case of a sidecar, it says, “I’m on Kubernetes, I’m a Docker container, and my role is an nginx service.” The agents subscribe to topics, called subscriptions, on the backend in order to gather data. Sensu supports a wide variety of plugins, from [Nagios plugins](https://blog.sensu.io/the-story-of-nagios-plugin-support-in-sensu) to your localhost [StatsD socket](https://docs.sensu.io/sensu-go/latest/reference/agent/#create-monitoring-events-using-the-statsd-listener) to [Prometheus endpoints](https://blog.sensu.io/the-sensu-prometheus-collector-972c441d45e) and more. 
+
+In this tutorial, we’ll configure a simple check with the Nagios `check_http` plugin.
+
+First, you need to provide the Sensu configuration, which includes four main attributes: resource type, API version, metadata, and spec. These get registered in the control plane and associate with the monitoring check configuration.
+
+For example, here’s a monitoring check in Sensu with the plugins needed to register that check:
+
+![Sensu monitoring check](https://user-images.githubusercontent.com/38439765/76876928-ab56a780-6840-11ea-84c4-c0e0555ff9d8.png)
+
+The output is in the format of `nagios_perfdata`, and I want it to write to InfluxDB.
+
+Let’s configure the monitoring check:
+
+```
+$ sensuctl --namespace webinar create -f sensu/checks/check-nginx.yaml
+```
+
+You’ll see the configuration appear in your Sensu dashboard, where it can be edited if needed, and it’ll automatically execute on all three of your Kubernetes pods. 
+
+![Sensu configuration in dashboard](https://user-images.githubusercontent.com/38439765/76877049-d04b1a80-6840-11ea-948e-c09e7a744739.png)
+
+Here’s your service check result:
+
+![Sensu check result in dashboard](https://user-images.githubusercontent.com/38439765/76877080-df31cd00-6840-11ea-8b51-447fcb66243b.png)
```

Comment:
For the live blog post, I think this image should move into the website repo and be served from there.